src/main/java/de/zalando/aruha/nakadi/NakadiRuntimeException.java

```diff
@@ -0,0 +1,19 @@
+package de.zalando.aruha.nakadi;
+
+import de.zalando.aruha.nakadi.domain.Problem;
+
+public class NakadiRuntimeException extends RuntimeException {
```

Comment:
To use it in places that will be used in streams and lambdas :)
Because the checked exception needed to be declared in functional interfaces.